<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QDialog" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>417</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Book title:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTitle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add a new book to library:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Author's name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtAuthorName">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ISBN:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtISBN">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtQuantity">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>240</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Quantity:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAddBook">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>310</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Javanese Text</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Saved books:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnShowBooks">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>310</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Show all books</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDelete">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>350</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Clear data</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listAllBooks">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>50</y>
     <width>431</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnList">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>350</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Activity list</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
